bin/swagger_style.py

```diff
@@ -19,6 +19,10 @@ def main() -> int:
     parser = argparse.ArgumentParser("Reads a correct swagger file and checks that it conforms to the style guide.")
     parser.add_argument("--swagger_path", help="path to the swagger file", required=True)
     parser.add_argument("--verbose", help="if set, prints as much information as possible.", action="store_true")
+    parser.add_argument(
+        "--with_line_number",
+        help="if set, prints the errors with the corresponding file name and line number.",
+        action="store_true")
     args = parser.parse_args()
 
     assert isinstance(args.swagger_path, str)
```

Comment:
I'd add an `assert isinstance(args.with_line_number, bool)` since there is no other way for mypy to know the type of the `args` properties.